app/assets/src/components/views/compare/SamplesHeatmapVis.jsx

```diff
@@ -147,13 +255,49 @@ class SamplesHeatmapVis extends React.Component {
             <DataTooltip data={this.state.nodeHoverInfo} />
           </div>
         )}
+        {this.state.columnMetadataHoverNode && (
+          <div
+            className={cx(
+              cs.tooltip,
+              this.state.columnMetadataHoverNode && cs.visible
+            )}
+            style={{
+              left: `${this.state.tooltipX + 20}px`,
+              top: `${this.state.tooltipY + 20}px`
+            }}
+          >
+            {this.renderColumnMetadataLegend(
+              this.state.columnMetadataHoverNode
+            )}
+          </div>
+        )}
+        {this.state.addMetadataTrigger && (
+          <ContextPlaceholder
```

Comment:
Could we make the styling of the `<ContextPlaceholder>` and columnMetadataLegend consistent? Specifically, the box-shadow and the border-radius? Right now, the `<ContextPlaceholder>` has no box-shadow, and no border-radius.

Reply:
You meant the search box...
i guess for this we have to decide what kind of element it is: dropdown? tooltip? :)
I will sync with Jenn tomorrow on how she sees this component.

Reply:
Ah yes, since the ContextPlaceholder is invisible.

Both of these elements "overlap" another element, so I think the box-shadow is appropriate. And it looks like most of our elements do have a small border-radius, which seems to tbe missing on the search box. But feel free to talk to Jenn.